indico/htdocs/js/indico/jquery/permissions_widget.js

```diff
@@ -91,10 +91,20 @@
             var self = this;
             var $permissions = $('<div>', {class: 'permissions-box flexrow f-a-center f-self-stretch'});
             var $permissionsList = $('<ul>').appendTo($permissions);
+            // When full access is enabled, always show read access
+            if (_.contains(permissions, '_full_access') && !_.contains(permissions, '_read_access')) {
```

Comment:
can we use `FULL_ACCESS_PERMISSION` / `READ_ACCESS_PERMISSION` constants here as well, just like we do in the python code?